x-pack/plugins/security_solution/cypress/tasks/detections/callouts.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const getCallOut = (id: string, options?: Cypress.Timeoutable) => {
+  return cy.get(`[data-test-subj="callout-${id}"]`, options);
```

Comment:
Please use the screens folder to define all the elements you are going to interact with.

Reply:
Done